### PR TITLE
WebGPU: Split out webgpu struct info

### PR DESCRIPTION
**Pull request by lokokung**
Splitting this out allows for easier maintainability.

### CODE REVIEW

File: src/struct_info_webgpu.json
Comment (sbc100) [inline src/struct_info_webgpu.json:4]: Maybe update the comment here?  "This file is ..."

File: system/include/webgpu/README.md
Comment (sbc100) [inline system/include/webgpu/README.md:24]: Perhaps just say "Copy .. to .. " here now?

**Review by sbc100 — APPROVED** [review]
Nice!

File: src/struct_info_webgpu.json
Comment (sbc100) [inline src/struct_info_webgpu.json:536]: Missing newline at EOF

File: src/struct_info_webgpu.json
Comment (lokokung) [reply src/struct_info_webgpu.json]: Done.

File: system/include/webgpu/README.md
Comment (lokokung) [reply system/include/webgpu/README.md]: Done.

File: src/struct_info_webgpu.json
Comment (lokokung) [reply src/struct_info_webgpu.json]: Done.

**Review by sbc100 — COMMENTED** [review]
lgtm.. but I'll wait for @kainino0x for final approval

**Review by kainino0x — APPROVED** [review]
LGTM, didn't realize we could do this! Makes updating much easier (even though we won't be doing rolling actively into Emscripten for a little while)

**Pull request merged**
